source/localizable/components/triggering-changes-with-actions.md

```diff
@@ -245,76 +245,90 @@ In this case, the code in `button-with-confirmation` does not change.
 It will still invoke `onConfirm` with no arguments.
 The action helper will add the arguments provided in the template to the call.
 
-Action arguments curry, meaning that you can provide partial arguments to the action helper and provide the rest of the
-arguments when you call the function within the component javascript file.
-For example, our `button-with-confirmation` component will now [yield](../wrapping-content-in-a-component/) the content
-of the confirmation dialog to collect extra information to be sent along with the `onConfirm` action:
+So far in our example,
+the action we have passed to `button-with-confirmation` is a function that accepts one argument, `messageType`.
+Suppose we want to extend this by allowing `sendMessage` to take a second argument,
+the actual text of the message the user is sending:
 
-```app/templates/components/button-with-confirmation.hbs
-<button {{action "launchConfirmDialog"}}>{{text}}</button>
-{{#if confirmShown}}
-  <div class="confirm-dialog">
-    {{yield confirmValue}}
-    <button class="confirm-submit" {{action "submitConfirm"}}>OK</button>
-    <button class="confirm-cancel" {{action "cancelConfirm"}}>Cancel</button>
-  </div>
-{{/if}}
+```app/components/send-message.js
+export default Ember.Component.extend({
+  actions: {
+    sendMessage(messageType, messageText) {
+      //send message here and return a promise
+    }
+  }
+});
```

Comment:
I like the reworking of the text that we've got here so far, as it simplifies the explanation by explaining what we're trying to achieve.